lib/std/net.zig

```diff
@@ -1724,13 +1722,15 @@ const ResolvConf = struct {
                 std.os.linux.IPV6.V6ONLY,
                 &mem.toBytes(@as(c_int, 0)),
             );
-            for (ns_list.items) |*ns| {
-                if (ns.any.family != posix.AF.INET) continue;
-                mem.writeInt(u32, ns.in6.sa.addr[12..], ns.in.sa.addr, native_endian);
-                ns.in6.sa.addr[0..12].* = "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff".*;
-                ns.any.family = posix.AF.INET6;
-                ns.in6.sa.flowinfo = 0;
-                ns.in6.sa.scope_id = 0;
+            for (&ns_arr) |*ns| {
+                if (ns.*) |*n| {
+                    if (n.*.any.family != posix.AF.INET) continue;
+                    mem.writeInt(u32, n.*.in6.sa.addr[12..], n.*.in.sa.addr, native_endian);
+                    n.*.in6.sa.addr[0..12].* = "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff".*;
+                    n.*.any.family = posix.AF.INET6;
+                    n.*.in6.sa.flowinfo = 0;
+                    n.*.in6.sa.scope_id = 0;
```

Comment:
One level pointers are dereferenced automatically so there is no need to extra `.*`.
```suggestion
                    if (n.any.family != posix.AF.INET) continue;
                    mem.writeInt(u32, n.in6.sa.addr[12..], n.in.sa.addr, native_endian);
                    n.in6.sa.addr[0..12].* = "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\xff".*;
                    n.any.family = posix.AF.INET6;
                    n.in6.sa.flowinfo = 0;
                    n.in6.sa.scope_id = 0;
```